docs/devel/architecture.md

```diff
@@ -0,0 +1,44 @@
+# karydia Concepts
+## Secure by Default
+The mission of karydia is to configure kubernetes in a more secure way. Many kubernetes default settings are unsecure. When karydia is installed these defaults are revised and supervised. Most karydia features can be switched on centrally and configured on namespace level.  
```

Comment:
As mentioned by @marwinski, `unsecure` may be an unfavorable wording. E.g. `Many kubernetes default settings are quite permissive`